openedx/stanford/djangoapps/auth_lagunita/tests/test_form.py

```diff
@@ -0,0 +1,11 @@
+# pylint: disable=missing-docstring
+from django.test import TestCase
+
+from ..forms import InfoForm
+
+
+class InfoFormTest(TestCase):
+
+    def test_basic(self):
```

Comment:
I think it's preferred to have form tests like in `openedx/stanford/djangoapps/register_cme/tests/test_form.py` one passing valid field data and one invalid

Reply:
That's true I guess but then it seems the only necessity to even have tests for Django forms is if they have more complex data validation. Do we simply need this trivial case to satisfy test coverage requirements?